skills/dev-backlog/references/file-format.md

```
@@ -37,42 +37,48 @@ Task files are compatible with the [Backlog.md](https://github.com/MrLesk/Backlo
 
 Extended set (if using more granular tracking):
 
-| Status | GitHub Label | Meaning |
+| Status | GitHub mapping | Meaning |
 |--------|-------------|---------|
 | `Blocked` | `status:blocked` | Waiting on external dependency |
 | `In Review` | `status:in-review` | PR under review |
 
 ## Filename Convention
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)